prow/repoowners/repoowners.go

```diff
@@ -707,6 +720,20 @@ func (o *RepoOwners) applyConfigToPath(path string, re *regexp.Regexp, config *C
 		}
 		o.labels[path][re] = sets.NewString(config.Labels...)
 	}
+	if len(config.SourcePath) > 0 {
+		if o.configSources[path] == nil {
+			o.configSources[path] = make(map[*regexp.Regexp]sets.String)
```

Comment:
style is generally `foo = map{}` rather than `foo = make(map)`

Reply:
I was going with the flow here; I'll check to see if I can take your suggestion on both my changes and in the changes above using the unit tests and report back.

Reply:
Ok, so I went nuclear here and replaced *all* the map make's with map literals in the file.

Unit tests still pass. Let me know if you are ok with this?